src/main/java/paymelah/model/person/NameComparator.java

```diff
@@ -0,0 +1,16 @@
+package paymelah.model.person;
+
+import java.util.Comparator;
+
+/**
+ * Compares a {@code Person} to another using the alphabetical ordering of their {@code Name}.
+ * Note: this comparator imposes orderings that are inconsistent with {@code equals} in {@code Person}.
+ */
+public class NameComparator implements Comparator<Person> {
+
+    @Override
+    public int compare(Person o1, Person o2) {
+        return o1.getName().compareTo(o2.getName());
```

Comment:
Does this violate the Law of Demeter design principle on only invoking methods from objects with a direct relation to it?

Reply:
Good point, I will add methods `compareNameWith` and `compareAmountOwedWith` to Person instead and make use of them in the Comparators.